### PR TITLE
Set RX Antenna

### PR DESCRIPTION
**Pull request by nsbruce**
The FlexSource can now check which RX Antenna is active as well as change the RX Ant port on the flex in use. The GRC block now has a dropdown which lets you pick which RX Ant to use. 

The only "maybe-bug" is that while the flowgraph is running, changing the RX Ant value doesn't seem to matter. So you can pick before launch which antenna to use but you can't change during operation.

### CODE REVIEW

File: python/FlexSource.py [hunk 1]
```diff
@@ -85,6 +85,23 @@ def set_bandwidth(self,bandwidth):
...
+
+    def set_rx_ant(self, rx_ant):
+        """
+        Sets the RX antenna of the underlying Panadapter
+
+        Args:
+            rx_ant: the new RX antenna to use
+        """
+        self._rx_ant = self.rx_ant
+        self.pan_adapter.RXAnt = self._rx_ant
```
Comment (cjam) [inline python/FlexSource.py:103]: Very nice, on the docstring comments.  Did someone get pylint up and running?

File: python/FlexSource.py [hunk 2]
```diff
@@ -123,12 +140,13 @@ def start(self):
         for p in pans:
             p.Close(True)
         self.pan_adapter = self.radio.GetOrCreatePanadapterSync(0, 0)
-        #self.pan_adapter.PropertyChanged += self.__property_changed
+        # self.pan_adapter.PropertyChanged += self.__property_changed
```
Comment (cjam) [inline python/FlexSource.py:143]: We would probably get rid of this now that we're done debugging things

**Review by cjam — APPROVED** [review]
Looks good man, nice work.

File: python/FlexSource.py (hunk 1 above)
Comment (nsbruce) [reply python/FlexSource.py:103]: No idea abut pylint haha. Just trying to stick with conventions.

File: python/FlexSource.py (hunk 2 above)
Comment (nsbruce) [reply python/FlexSource.py:143]: I still use it a bunch when testing new pieces. I'll tag it for removal when I haven't used it in a bit or have a different testing system setup.

**Pull request merged**